src/shell/dotnet/Shell/Shell.csproj

```diff
@@ -46,17 +46,37 @@
 		<ProjectReference Include="..\..\..\messaging\dotnet\src\Host\MorganStanley.ComposeUI.Messaging.Host.csproj" />
 	</ItemGroup>
 
-	<PropertyGroup>
-		<Fdc3BundleOutput>$(ComposeUIRepositoryRoot)/src/shell/js/composeui-fdc3/dist/fdc3-iife-bundle.js</Fdc3BundleOutput>
-	</PropertyGroup>
+	<Target Name="InstallMonoRepoDependencies"
+		Condition="!Exists('$(ComposeUIRepositoryRoot)\node_modules')">
+		<Message Importance="high" Text="Installing node_modules..." />
+		<Exec Command="cd $(ComposeUIRepositoryRoot) | npm install" />
```

Comment:
Are you sure about the pipe characters (here and in other scripts in this file)? That means redirect output. If you mean "do A, and if successful then B" then use `&&`:
```
cd $(ComposeUIRepositoryRoot) && npm install
```